recipes/parallel-numpy-rng/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "parallel-numpy-rng" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/parallel-numpy-rng-{{ version }}.tar.gz
+  sha256: 8d7b101820e2def248cd0b6f30f4171886d98f8291c65821c41a7109cef52c36
+
+build:
+  skip: true  # [win]
```

Comment:
Why are you skipping Windows? Can this one be noarch?

Reply:
@ocefpaf Thanks for the suggestion - I converted the recipe to be `noarch`.